src/utils/helpers/browser/browser.js

```diff
@@ -60,6 +60,16 @@ const Browser = {
   },
 
   /**
+   * Focues and sets cursor of input filed but does not select text
+   *
+   * @method setFocus
+   */
+  setFocus: (ref) => {
+    let node = ReactDOM.findDOMNode(ref._input);
```

Comment:
Or maybe you should send in the entire thing that you want the dom node of rather than the function knowing the internal of some of the components.